### ISSUE TITLE
Notebook 15 - Multiple Reactions at Equilibrium

### ISSUE THREAD

**Issue by joeegitto22** [dpploy/chen-3170#23]
There is a lot of complex math behind all of this code and I would recommend spending a little bit more time on it if you can to make sure students fully understand it. Once the first few blocks of code are described (based on the math), it does help clarify things and make the information easier to apply to a real example.

**Comment by dealmeidavf** [comment]
Right. A note to myself. This notebook will have to be split into two so globally convergent Newton's method is introduced; almost a necessity here when the following notebook, treating temperature and pressure dependent equilibria, is covered. In addition, at this point, import of notebooks will be necessary to clean up the programming. This will be done in the future.
